linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_csh_cshrc/rule.yml

```diff
@@ -37,20 +37,23 @@ references:
     stigid@ol8: OL08-00-020353
     stigid@rhel8: RHEL-08-020353
 
-ocil_clause: 'the above command returns no output, or the umask is configured incorrectly'
+ocil_clause: 'the value for the "umask" parameter is not "{{{ xccdf_value("var_accounts_user_umask") }}}", or the "umask" parameter is missing or is commented out'
 
 ocil: |-
-    Verify the <tt>umask</tt> setting is configured correctly in the <tt>/etc/csh.cshrc</tt> file by
-    running the following command:
-    <pre>$ sudo grep "umask" /etc/csh.cshrc</pre>
-    All output must show the value of <tt>umask</tt> set as shown in the below:
-    <pre>umask {{{ xccdf_value("var_accounts_user_umask") }}}</pre>
+    Verify the "umask" setting is configured correctly in the "/etc/csh.cshrc" file with the following command:
+
+    Note: If the value of the "umask" parameter is set to "000" "/etc/csh.cshrc" file, the Severity is raised to a CAT I.
```

Comment:
With this specific STIG language should we move this content to the `checktext` key?